src/dcat-us/dataset-formatter.test.ts

```diff
@@ -342,7 +342,7 @@ describe('formatDcatDataset', () => {
     const expected = {
       '@type': 'dcat:Dataset',
       identifier: `${siteUrl}/datasets/00000000000000000000000000000000_0`,
-      license: null,
```

Comment:
@drspacemanphd I changed license to be null instead of '' for an issue a couple months ago. Are we sure we want to go back?

Reply:
After reviewing, the switch to null was a style preference on Tate's part, not a product requirement. I'm fine to leave this.

Reply:
@sonofflynn89 @drspacemanphd 

It's not really a style thing on my part. It all comes down to the DCAT harvesters. Thomas told me that they seemed to prefer `null`, so I tried to be consistent with that. If it doesn't matter, this is fine with me!

Another thing to be aware of is that when you have a JSON-LD node, sometimes the entire node should be removed when one of the values isn't available.

E.g.
```json
"foo": {
  "@id": "dct:somekey",
  "importantValue": null
}
```

I think `adlib` can do this by means of the ["Optional Transform."](https://github.com/Esri/adlib#optional-transform).